tests/BlazorServerCallsGraph/Pages/CallWebApi.razor

```diff
@@ -20,13 +20,25 @@ else
 }
 
 @code {
-    private string apiResult;
+    private HttpResponseMessage response;
+    string apiResult;
 
     protected override async Task OnInitializedAsync()
     {
         try
         {
-            apiResult = await downstreamAPI.CallWebApiAsync("me");
+            response = await downstreamAPI.CallWebApiForUserAsync(
+                    "CalledApi",
```

Comment:
Here we call the service "CalledApi", on behalf of the user (this maps to some configuration for this service: see below), and we override the relative path to call the "me" endpoint. All the other configurations are overridable as well.